planetarypraxis/smartforests
# Add Posthog analytics

- [X] Add Posthog to the public site and the CMS
- [X] Identify logged in users
- [X] Set up PostHog hosted appropriately 
- [X] Add `POSTHOG_PUBLIC_TOKEN`  to Digital Ocean env

If there are any particular events we feel we want to track, comment here and we can sort it out:

- [x] Wagtail publishing events (page creation, revision, move to new location, moderation, etc.)
- [x] Django auth events (registration, sign in, sign out, etc.)

### Testing

- Comment out "opt_out..." in `smartforests/templates/posthog/initialise.html` BEFORE running the system, or else you will be permanently opted out. If you have already been opted out, you can run `posthog.opt_in_capturing()` once to test again.
- Add `POSTHOG_PUBLIC_TOKEN` to your `smartforests/settings/local.py`
- Either ask Jan or get the token from https://app.posthog.com/project/settings (Lastpass for details, hello@commonknowledge.coop is the username; then select the Planetary Practice org -> Smart Forest project)
- Check the app.posthog.com events panel to see events being tracked.
